pytorch_lightning/utilities/cloud_io.py

```diff
@@ -12,15 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import io
 from distutils.version import LooseVersion
 from pathlib import Path
 from typing import IO, Union
 
 import fsspec
+from fsspec.implementations.local import LocalFileSystem
+
 import torch
 
 
+class LightningLocalFileSystem(LocalFileSystem):
```

Comment:
Can we make this protected in case this stays here for a while?

```suggestion
class _LightningLocalFileSystem(LocalFileSystem):
```